monkey/infection_monkey/readme.md

```diff
@@ -1,26 +1,26 @@
-To get development versions of Monkey Island and Monkey look into deployment scripts folder.
-If you only want to build monkey from scratch you may reference instructions below.
+>To easily setup development environment for Monkey Island and the Monkey look into deployment scripts folder.
+
+>If you want to setup dev. env. for the Monkey manually, refer to the instructions below.
+
 
 The monkey is composed of three separate parts.
 * The Infection Monkey itself - PyInstaller compressed python archives
 * Sambacry binaries - Two linux binaries, 32/64 bit.
 * Mimikatz binaries - Two windows binaries, 32/64 bit.
 * Traceroute binaries - Two linux binaries, 32/64bit.
 
---- Windows ---
+## Windows
 
-1. Install python 2.7.15
-    Download and install from: https://www.python.org/downloads/release/python-2715/
-2.	Add python directories to PATH environment variable (if you didn't install ActiveState Python)
-	a. Run the following command on a cmd console (Replace C:\Python27 with your python directory if it's different)
-		setx /M PATH "%PATH%;C:\Python27;C:\Python27\Scripts
-	b. Close the console, make sure you execute all commands in a new cmd console from now on.
-3.	Install further dependencies
-	a. install VCForPython27.msi
-		https://aka.ms/vcpython27
-	b. if not installed, install Microsoft Visual C++ 2010 SP1 Redistributable Package
-		32bit: http://www.microsoft.com/en-us/download/details.aspx?id=8328
-		64bit: http://www.microsoft.com/en-us/download/details.aspx?id=13523
+1. Install python 3.7.4
+    Download and install from: https://www.python.org/ftp/python/3.7.4/
+2. Add python directories to PATH environment variable
+    1. Run the following command on a cmd console (Replace C:\Python37 with your python directory if it's different) 
+    `setx /M PATH "%PATH%;C:\Python37;C:\Python37\Scripts`
```

Comment:
This isn't part of the install? I think it is, or at least, optional

Reply:
IDK if it adds scripts folder.

Reply:
Please check, I think it does then this is useless

Reply:
Checked. Yes it does. I've changed it a bit, but I don't think this is useless, if user forgets the option or if he already have py3 installed, so I left these commands